.github/workflows/review-feedback-loop.yml

```
@@ -3,6 +3,22 @@ name: Review Feedback Loop
+  # witness, or a Copilot follow-up. So a PR could sit `blocked` with a stale
+  # review/threads-open label until an unrelated push/reopen happened to nudge
+  # it. Wiring pull_request_review_thread closes that gap: resolving the last
+  # thread recomputes state and arms an eligible PR immediately.
+  #
+  # Safe the same way review-response.yml (pull_request_review) already is: this
+  # event runs the workflow from the trusted DEFAULT branch and the job checks
+  # out the default branch, so no untrusted PR-head code executes. Arming is not
+  # merging — branch protection still gates the actual merge. Recursion is
+  # bounded: when sync-pr resolves an outdated thread it uses GITHUB_TOKEN, and
+  # GitHub does not spawn new workflow runs from GITHUB_TOKEN-driven events.
+  pull_request_review_thread:
```

actionlint (1.7.12)

[error] unknown Webhook event "pull_request_review_thread". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)